protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -248,6 +248,9 @@ var (
 	handleSend := func() error {
 		var protoReq {{.Method.RequestType.GoType .Method.Service.File.GoPkg.Path}}
 		err = dec.Decode(&protoReq)
+		if err == io.EOF {
```

Comment:
Code parity is fine.

But I am wondering if it falls into https://github.com/tmc/grpc-gateway/blob/b069c864828f47dac69037b625ae20953d4cd430/examples/examplepb/stream.pb.gw.go#L245 even though it is valid to send a zero-length sequence.